htdocs/projet/tasks/list.php

```diff
@@ -988,7 +988,7 @@
 		// Description
 		if (!empty($arrayfields['t.description']['checked'])) {
 			print '<td>';
-			print dol_escape_htmltag($object->description);
+			print nl2br($object->description);
```

Comment:
For content that can have several lines into a list, i recommend to use instead
dolGetFirstLineOfText($object->description, 5)
to restrict to 5 first lines only.